models/index.js

```diff
@@ -0,0 +1,16 @@
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize('users', 'root', 'root123', {
```

Comment:
instead of using sequelize details directly use env and move all the details in env file.